api/docs/v2/storefront/index.yaml

```diff
@@ -5380,6 +5399,14 @@ components:
         - state_name
         - country_iso
   parameters:
+    AddressId:
```

Comment:
@pmilewski please also use this paremeter in line 265 to describe address update